sdk/nestjs/src/VariableValue.ts

```diff
@@ -5,23 +5,31 @@ import {
     DevCycleUser,
 } from '@devcycle/nodejs-server-sdk'
 import { ClsServiceManager } from 'nestjs-cls'
+import { VariableDefinitions } from '@devcycle/types'
 
-type VariableValueData = {
-    key: string
-    default: DVCVariableValue
+export type VariableValueData<
```

Comment:
does this appropriately handle for new keys that are added? do we need to add or update any tests to valdiate?

Reply:
I'm not sure how we'd make a test for that